README.md

```diff
@@ -16,18 +16,15 @@ Wakeword voice service for MALOS. The last version support:
 
 ### Raspbian Dependencies 
 
-Before, please install FPGA and MCU drivers on your RaspberryPi3 and perform device reboot. 
+Before, please install MALOS on your RaspberryPi3 and perform device reboot. For more details: [Getting Started Guide](https://github.com/matrix-io/matrix-creator-quickstart/wiki/2.-Getting-Started)
 
 ``` 
 echo "deb http://packages.matrix.one/matrix-creator/ ./" | sudo tee --append /etc/apt/sources.list
-sudo apt-get clean
 sudo apt-get update
 sudo apt-get upgrade
-sudo apt-get install matrix-creator-init wiringpi cmake g++ git libzmq3-dev --no-install-recommends
+sudo apt-get install matrix-creator-init matrix-creator-malos cmake g++ git libzmq3-dev --no-install-recommends
```

Comment:
For now please restore installation flow, because we need first wiringpi and matrix-creator-init only from stable channel and then from unstable channel matrix-crator-malos-wakeword